libs/features/personalization/personalization.js

```diff
@@ -655,8 +655,9 @@ function parsePlaceholders(placeholders, config, selectedVariantName = '') {
     'value',
     'other',
   ];
-  const [val] = Object.entries(placeholders[0])
-    .find(([key]) => valueNames.includes(key.toLowerCase()));
+  const keys = placeholders && placeholders.length ? Object.entries(placeholders[0]) : [];
```

Comment:
Can we long term try to harmonize this? This seems to rebuild the existing milo placeholder functionality, it'd be better if you can use the core placeholders methods and extend them to support MEP use-cases where needed.

We shouldn't have two different places of code that do the same thing.

Reply:
@mokimo I agree this is something we should keep an eye on.  If the functionalityfor MEP in the existing placeholders feature becomes larger, then I do think we want will want to use export to share the relevant code.  But right now, there is only one line of code in placeholders to use MEP. 

In `libs\features\placeholders.js` in the `getPlaceholder` function, there is this line:
`if (config.placeholders?.[key]) return config.placeholders[key];`

And that's it.  So I don't think it makes sense to do it at this point.

Reply:
Keeping an eye on it sounds good and I'm fine to keep this as they are for now 👍